### DISCUSSION TITLE
Welcome to nephio Discussions!

### DISCUSSION THREAD

**Discussion by johnbelamaric** [nephio-project/nephio#407] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by johnbelamaric** [comment]
@liamfallon I enabled this, let me know if you need anything else.

**Reply by liamfallon** [reply]
Thanks @johnbelamaric! Let's use it for a while and see how it works out.
